tutorials/simulation_exporting.ipynb

```diff
@@ -0,0 +1,326 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.            # FIXME: EK, should we rather use the dof manager for this?
</pre><p>Yes, I believe that would be cleaner.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/708/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
On second thought, that would require two steps: Extracting u_j and reconstructing jump on fractures. Furthermore, the current solution generalizes to any (compound) ad operator. I'll leave it as is